scriptmodules/emulators/amiberry.sh

```diff
@@ -106,12 +106,15 @@ function configure_amiberry() {
 
     # move config / save folders to $md_conf_root/amiga/amiberry
     local dir
-    for dir in conf savestates screenshots; do
+    for dir in conf nvram savestates screenshots; do
         moveConfigDir "$md_inst/$dir" "$md_conf_root/amiga/amiberry/$dir"
     done
 
-    # symlink cd32.nvr from $md_inst/data to "$md_conf_root/amiga/amiberry
-    moveConfigFile "$md_inst/data/cd32.nvr" "$md_conf_root/amiga/amiberry/cd32.nvr"
+    # check for cd32.nvr and move it to $md_conf_root/amiga/amiberry/nvram
+    if [[ -f "$md_conf_root/amiga/amiberry/cd32.nvr" ]]; then
+	mv "$md_conf_root/amiga/amiberry/cd32.nvr" "$md_conf_root/amiga/amiberry/nvram/"
+	rm "$md_inst/data/cd32.nvr"
```

Comment:
This line shouldn't be needed I think?

Reply:
Still needed. No later than yesterday a user complained about it (see [ticket](https://github.com/BlitterStudio/amiberry/issues/1044)) :) Any reason why you ask?

Reply:
Previously the file was moved and symlinked to "$md_conf_root/amiga/amiberry/cd32.nvr". "$md_inst" is removed on install.

Reply:
In the ticket it looks like they are installing manually rather than via the RetroPie-Setup script since we don't include v5.5 yet. But in any case `$md_inst` is wiped on install so there shouldn't be any file there unless you are including it with amiberry in that location.

Reply:
Just to confirm as I just tested - did you test this with a file in place at `"$md_conf_root/amiga/amiberry/cd32.nvr"`. You will get this error:

```
= = = = = = = = = = = = = = = = = = = = =
Configuring 'amiberry' : Amiga emulator with JIT support (forked from uae4arm)
= = = = = = = = = = = = = = = = = = = = =

rm: cannot remove '/opt/retropie/emulators/amiberry/data/cd32.nvr': No such file or directory
/home/pi/RetroPie-Setup
```

Reply:
Got it. You were exactly referring to:

```
rm "$md_inst/data/cd32.nvr"
```

As I didn't read properly I thought you talked about the whole `if` block 😢 After a good night of sleep I removed the line above and test is ok. Apologies for the time waste...